wordpress-mobile/WordPress-Android
# Send invites to WordPress.com sites

Related to #3901 

Adds the ability to send invites to users to become members of the site.

To test:
* Go to "People" from the Site screen
* Tap the "+" button in the top-right
* Start typing usernames or emails in the top field
* Use "Enter" to input multiple usernames/emails. Focusing away from the input field should also "register" the username
* Usernames/email will turn red if there is any issue with them
* (Optional) Select a role
* (Optional) Enter a custom message to be send with the invite
* Tap "SEND" to send the invite. Send will stop if there are any invalid usernames/emails
* Screen will "clear" if SEND succeeds


Needs review: @bummytime
